libraries/stdlib/jvm/test/text/StringJVMTest.kt

```diff
@@ -75,53 +75,67 @@ class StringJVMTest {
     }
 
     @Test fun capitalize() {
+        fun testCapitalize(expected: String, string: String) {
+            assertEquals(expected, string.capitalize())
+            assertEquals(expected, string.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() })
```

Comment:
In general, these are not equivalent because `capitalize` depends on locale and `titlecase` does not.
Better to use the equivalent replacement as proposed in KEEP.

Reply:
done